### PR TITLE
Add mahotas

### PR DESCRIPTION
**Pull request by jakirkham**
Adds mahotas, a computer vision library. Recipes borrowed from conda recipes. Cleaned up to be a bit nicer and match the style here at conda forge. Builds on all platforms.

### CODE REVIEW

File: recipes/mahotas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```
Comment (ocefpaf) [inline recipes/mahotas/meta.yaml]: Don't know mahotas, but I am assuming it depends on numpy's ABI, right?

Comment (jakirkham) [inline recipes/mahotas/meta.yaml]: Yes, it links against it. Has C++ code that does this.

**Comment by jakirkham** [comment]
Passing. :)

Any more suggestions?

**Comment by jakirkham** [comment]
Just wanted to let you know that we are adding `mahotas` to `conda`, @luispedro. Thanks for this great package. This makes it easier to install and work with packages like yours that have source code and may depend on things like `numpy`. If you notice anything that is off or are making new release, feel free to ping me. Also, if you would like to help out on maintenance (even in a minor way), I would be happy to add you as a maintainer.

**Comment by pelson** [comment]
For the record, the maintained recipe has been moved to https://github.com/conda-forge/mahotas-feedstock, and the permissions on the repository are completely controlled by the recipe maintainers in https://github.com/conda-forge/mahotas-feedstock/blob/master/recipe/meta.yaml#L36.

**Comment by luispedro** [comment]
@jakirkham Thanks for letting me know. I will ping you (or submit a PR) for new versions.

**Comment by jakirkham** [comment]
Excellent, thanks @luispedro.

**Pull request merged**